### DISCUSSION TITLE
What about language support ?

### DISCUSSION THREAD

**Discussion by AlexandreRousseau36** [tneotia/html-editor-enhanced#251] [Ideas]
Hey,
This package is really nice. Thank you for your work.
But I can't find a way to change the language of the dialogs like the color picker dialog.

![2022-05-10 13 54 50](https://user-images.githubusercontent.com/15885534/167622831-ab49b036-e387-4b23-a170-3dd62fdc1f6d.jpg)

Will this be possible or is it already doable ?

Regards.

**Comment by tneotia** [comment]
The current solution is https://github.com/tneotia/html-editor-enhanced/issues/69

I'd like to add a better API for translation, but just haven't had much time for this project. Don't know when things will free up, definitely not before August / September.
